server/game/cards/07.2-FP/MartyrsCry.js

```diff
@@ -0,0 +1,29 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class MartyrsCry extends ActionCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Resolution: Martyr\'s Cry',
+            playType: 'resolution',
+            cost: ability.costs.discardFromPlay({
+                type: 'spell',
+                condition: card => card.isMiracle() && card.parent.isParticipating()

```

Comment:
Also check if there is `card.parent` to be absolutely sure.

Reply:
Notably, this part came from Hex Slingin', which does not have that check.